memcached/init.lua

```diff
@@ -386,7 +410,20 @@ local function memcached_get(name)
     return memcached_services[name]
 end
 
+local function memcached_slab()
+    local slab_arena_info = memcached_internal.memcached_slab_arena_info()
+    local slab = {
+        info = {}
+    }
+    for _, v in pairs(slab_arena_info_table) do
+	slab.info[v] = slab_arena_info[0][v]
```

Comment:
It seems, a tab symbol is here (or indent is wrong due to some other reason).

Reply:
Fixed.

```lua
--- a/memcached/init.lua
+++ b/memcached/init.lua
@@ -416,7 +416,7 @@ local function memcached_slab()
         info = {}
     }
     for _, v in pairs(slab_arena_info_table) do
-       slab.info[v] = slab_arena_info[0][v]
+        slab.info[v] = slab_arena_info[0][v]
     end
 
     return slab
```